src/Traits/Body/ChecksForWithBody.php

```diff
@@ -5,7 +5,7 @@
 namespace Saloon\Traits\Body;
 
 use Saloon\Http\PendingRequest;
-use Saloon\Contracts\Body\WithBody;
+use Saloon\Contracts\Body\HasBody;
 use Saloon\Exceptions\BodyException;
 
 trait ChecksForWithBody
```

Comment:
The interface was renamed to HasBody, which it now checks for.

```suggestion
trait ChecksForHasBody
```